examples/p4est_2d_dgsem/elixir_navierstokes_NACA0012airfoil_mach08.jl

```diff
@@ -142,14 +141,29 @@ lift_coefficient_shear_force = AnalysisSurfaceIntegral(semi, force_boundary_name
                                                                                   u_inf(equations),
                                                                                   l_inf()))
 
+# TODO - Should these two be moved to a different callback to avoid printing "Dummy Value" so often?
+# This will be fixed if the "Dummy Callback" is somehow not printed at all.
+# Can this be done by overloading on `pretty_form_utf`?
+friction_coefficient = AnalysisSurface(semi, force_boundary_names,
+                                       SurfaceFrictionCoefficient(rho_inf(),
+                                                                  u_inf(equations),
+                                                                  l_inf()))
+
+pressure_coefficient = AnalysisSurface(semi, force_boundary_names,
+                                       SurfacePressureCoefficient(p_inf(), rho_inf(),
+                                                                  u_inf(equations),
+                                                                  l_inf()))
+
 analysis_callback = AnalysisCallback(semi, interval = analysis_interval,
                                      output_directory = "out",
                                      save_analysis = true,
                                      analysis_errors = Symbol[],
                                      analysis_integrals = (drag_coefficient,
                                                            lift_coefficient,
                                                            drag_coefficient_shear_force,
-                                                           lift_coefficient_shear_force))
+                                                           lift_coefficient_shear_force,
+                                                           friction_coefficient,
+                                                           pressure_coefficient))
```

Comment:
We should implement this in some fashion that this is not treated/labeled as an integral to avoid confusion.